gpu_bdb/bdb_tools/utils.py

```diff
@@ -299,7 +299,7 @@ def run_dask_cudf_query(config, client, query_func, write_func=write_result):
         benchmark(
             write_func,
             results,
-            output_directory=config["output_dir"],
+            output_directory=config["output_dir"] or "./",
```

Comment:
I think we should we just handle `output_dir ` in the `argparser` directly in 1 place  [here](https://github.com/rapidsai/gpu-bdb/blob/7847b35cb43ae086d2173e05916f230999b33898/gpu_bdb/bdb_tools/utils.py#L387) through a check  rather than doing it here.